lms/djangoapps/instructor/static/instructor/ProblemBrowser/components/BlockBrowser/BlockBrowser.jsx

```diff
@@ -0,0 +1,58 @@
+/* global gettext */
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import * as classNames from 'classnames';
+import {Button} from '@edx/paragon';
+
+const FAIcon = ({iconName, altlabel}) => (
+    <div>
+        <span className={classNames('icon fa', `fa-${iconName}`)} aria-hidden="true"/>
+        <span className="sr">{altlabel}</span>
+    </div>
+);
```

Comment:
Any chance I could convince you to give paragon's `Icon` component a try here?

[Storybook link](https://edx.github.io/paragon/?selectedKind=Icon&selectedStory=with%20screenreader%20text&full=0&addons=1&stories=1&panelRight=1&addonPanel=storybook%2Factions%2Factions-panel)